src/main/java/common/Winner.java

```diff
@@ -0,0 +1,20 @@
+package common;
+
+public enum Winner {
+        PLAYER(0),
+        COMPUTER(1),
+        DRAW(2);
+
+        private int code;
+
+
+    Winner(int code) {
+        this.code = code;
+    }
+
+    public int getCode() {
```

Comment:
This method is never used. Do you need `code` at all? If not, why not create a simpler enum:
```
public enum Winner {
        PLAYER,
        COMPUTER,
        DRAW
}
```
Same applies to the `Outcomes` class in this package